File: Swerve Java/src/org/team537/Robot.java
package org.team537;

import edu.wpi.first.wpilibj.IterativeRobot;

public class Robot extends IterativeRobot {
	private Swerve swerve;

	@Override
	public void robotInit() {
		swerve = new Swerve();
	}

	@Override
	public void autonomousPeriodic() {
	}

	@Override
	public void teleopPeriodic() {
		swerve.drive();
		// swerve.tune(swerve.frontRightAngle);
		swerve.dashboard();
	}

	@Override
	public void testPeriodic() {
	}
}

File: Swerve 2.0/src/Robot.cpp
#include "Schematic.hpp"
#include "Swerve.hpp"

class Robot: public IterativeRobot {
private:
	Joystick *joystick;
	Gyro *gyro;
	Swerve *swerve;

	void RobotInit() {
		joystick = new Joystick(0);
		gyro = new Gyro(1);
		gyro->Reset();
		swerve = new Swerve(joystick, gyro);
	}

	void AutonomousInit() {
	}

	void AutonomousPeriodic() {
	}

	void TeleopInit() {
	}

	void TeleopPeriodic() {
		swerve->Drive();
		// swerve->Tune(swerve->frontRightAngle);
		// swerve->TuneMinMax(swerve->frontRightAngle);
		swerve->Dashboard();
	}

	void TestPeriodic() {
	}
};

START_ROBOT_CLASS(Robot);
